qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/NavigationViewFragment.kt

```diff
@@ -8,7 +8,8 @@ import androidx.fragment.app.Fragment
 import com.mapbox.navigation.dropin.NavigationView
 import com.mapbox.navigation.dropin.NavigationViewOptions
 import com.mapbox.navigation.dropin.ViewProvider
-import com.mapbox.navigation.examples.core.databinding.FragmentNavigationViewBinding
+import com.mapbox.navigation.qa_test_app.R
+import com.mapbox.navigation.qa_test_app.databinding.LayoutFragmentNavigationViewBinding
```

Comment:
Looking at the code more closely, I just remembered that this whole fragment is using the old dropInUI related API(s). This fragment code needs to be changed and now it should be as simple as

```
class NavigationViewFragment : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        val binding = LayoutFragmentNavigationViewBinding.inflate(inflater)
        return binding.root
    }
}
```